### PR TITLE
Fixes #11113: Save button is not disable during saving

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://www.rudder-project.org/redmine/issues/11113

### CODE REVIEW

File: builder/js/ncf.js [hunk 1]
```diff
@@ -191,6 +191,7 @@ app.controller('ncf-builder', function ($scope, $modal, $http, $log, $location,
   // Open/Close by default the Conditions box
   $scope.conditionIsOpen = false;
 
+  $scope.saving = "ok";
```
Comment (VinceMacBuche) [inline builder/js/ncf.js]: What is the usage of $scope.saving ?

Comment (VinceMacBuche) [inline builder/js/ncf.js]: Why "ok" and not "false" ?

Comment (RaphaelGauthier) [inline builder/js/ncf.js]: Oops, my bad. It should be _false_ !!

**Comment by RaphaelGauthier** [comment]
Commit modified

**Comment by RaphaelGauthier** [comment]
PR rebased

**Pull request merged**